### ISSUE TITLE
aiohttp.ClientSession() crashes pylint

### ISSUE THREAD

**Issue by ppickfor** [PyCQA/pylint#8509]
### Bug description

import aiohttp
s = aiohttp.ClientSession() 
[pylint-crash-2023-03-28-20-44-10.txt](https://github.com/PyCQA/pylint/files/11096417/pylint-crash-2023-03-28-20-44-10.txt)

### Command used

```shell
pylint test.py
```

### Pylint output

```shell
see attached
```

### Expected behavior

not to crash

### Pylint version

```shell
pylint 2.17.1
astroid 2.15.1
Python 3.11.2 (tags/v3.11.2:878ead1, Feb  7 2023, 16:38:35) [MSC v.1934 64 bit (AMD64)]
```

### OS / Environment

Windows

### Additional dependencies

aiodns==3.0.0
aiohttp==3.8.4
aiosignal==1.3.1
astroid==2.15.1
async-timeout==4.0.2
attrs==22.2.0
autopep8==2.0.2
certifi==2022.12.7
cffi==1.15.1
charset-normalizer==3.1.0
colorama==0.4.6
dill==0.3.6
frozenlist==1.3.3
idna==3.4
isort==5.12.0
lazy-object-proxy==1.9.0
mccabe==0.7.0
multidict==6.0.4
mypy==1.1.1
mypy-extensions==1.0.0
platformdirs==3.2.0
pycares==4.3.0
pycodestyle==2.10.0
pycparser==2.21
pylint==2.17.1
requests==2.28.2
tomlkit==0.11.7
types-requests==2.28.11.17
types-urllib3==1.26.25.10
typing_extensions==4.5.0
urllib3==1.26.15
wrapt==1.15.0
yarl==1.8.2

**Comment by DanielNoord** [comment]
Please copy the content of the stack trace into the issue description. Maintainers are not likely to download random files from the internet because of the apparent security risks :smile:

**Comment by Pierre-Sassoulas** [comment]
Thank you for opening the issue :) Don't worry, I included the stacktrace in the issue.

**Comment by DanielNoord** [comment]
@jacobtylerwalls Are you able to reproduce this?

I can't:

```python
❯ cat test.py
import aiohttp

s = aiohttp.ClientSession()
❯ pylint --version
pylint 3.0.0b1
astroid 2.16.0dev0
Python 3.11.0b3 (main, Jun 12 2022, 16:13:11) [Clang 13.0.0 (clang-1300.0.29.30)]
❯ pylint test.py

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
```

**Comment by jacobtylerwalls** [comment]
No, I can't. But the stack trace looks like something important to fix, somewhere downstream of the namedtuple/enum brain an Uninferable got attached to the `.args` of an `Arguments` node, which shouldn't have happened. We could move this to astroid, I guess.

**Comment by DanielNoord** [comment]
Yeah, although that might also be a downstream package breaking something. Without an actual reproducer this will just stay open indefinitely..
